ember/app/components/circling-performance-table.js

```diff
@@ -1,14 +1,10 @@
 import Ember from 'ember';
+import { findBy } from 'ember-awesome-macros/array';
+import raw from 'ember-macro-helpers/raw';
 
 export default Ember.Component.extend({
-  left: computedFindBy('perf', 'circlingDirection', 'left'),
-  right: computedFindBy('perf', 'circlingDirection', 'right'),
-  mixed: computedFindBy('perf', 'circlingDirection', 'mixed'),
-  total: computedFindBy('perf', 'circlingDirection', 'total'),
+  left: findBy('perf', raw('circlingDirection'), raw('left')),
```

Comment:
These are suffering from the current bug where `perf.@each.circlingDirection` is not being watched for changes. Should be fixed soon.